### PR TITLE
fix(slider): hide ticks when slider is disabled

### PR DESCRIPTION
**Pull request by DevVersion**
* No longer shows the ticks on hover when the slider is disabled.

### CODE REVIEW

File: src/lib/slider/_slider-theme.scss [hunk 1]
```diff
@@ -5,7 +5,6 @@
 @mixin md-slider-theme($theme) {
   $accent: map-get($theme, accent);
 
-  // TODO(iveysaur): Find an implementation to hide the track under a disabled thumb.
```
Comment (jelbourn) [inline src/lib/slider/_slider-theme.scss]: This `TODO` was actually referring to something else. If you look at the spec, you'll see that there's a tiny empty space around the thumb that covers up the track when the slider is disabled; this is talking about that.

Comment (DevVersion) [inline src/lib/slider/_slider-theme.scss]: Isn't that already implemented? (I probably miss something)

Comment (jelbourn) [inline src/lib/slider/_slider-theme.scss]: Oh, I didn't realize.

**Comment by jelbourn** [comment]
LGTM

**Comment by mmalerba** [comment]
It looks like this causes non-disabled sliders to not render the last tick. Make sure to rebase on top of #2641 it adds some more corner cases to watch out for.

Please include me as a reviewer on slider PRs in the future.

**Comment by DevVersion** [comment]
@mmalerba Yeah I locally tested this PR with your PR #2641 but at this time it wasn't merged yet.

Also from looking at the diff, there hasn't been any change for tick sequencing. It only hides the `.md-slider-ticks` class when a slider is disabled?

And yes can assign you to future slider PR's

**Comment by mmalerba** [comment]
I think the css rule precedence started getting unwieldy so I added a `md-hide-last-tick` class, you could probably just add `|| disabled` to the condition for that class

**Comment by DevVersion** [comment]
@mmalerba Updated everything as discussed on Slack.

**Pull request merged**